### DISCUSSION TITLE
训练图节点路由不可用: 138_node1->alice

### DISCUSSION THREAD

**Discussion by lllabmaster** [secretflow/secretflow#910] [Q&A]
新增加了一个节点，但是运行时出现以上错误，看不到详细日志。

138_node1 是另外一台机器，我把bob的数据集上传上去了，想测下跨机器访问，两台机器都是同一个内网，网络是通的。

节点状态：正常
数据集：正常（/home/kuscia/var/storage/data  都是有数据的，通过平台页面配置的数据）

**Comment by zhouaihui** [comment]
hi, @lllabmaster, 看起来是kuscia相关问题，请移步到kuscia下面提交issue。
